RUFAS/data_structures/animal_to_manure_connection.py

```diff
@@ -0,0 +1,217 @@
+from dataclasses import dataclass
+from enum import Enum
+from RUFAS.enums import AnimalCombination
+
+
+class ManureStreamType(Enum):
```

Comment:
```suggestion
class StreamType(Enum):
```
I suggest just calling this "StreamType". I think it is a little confusing to call this "ManureStreamType" because it isn't an attribute of the "ManureStream" object